core/src/main/java/org/prevayler/TransactionBase.java

```diff
@@ -0,0 +1,19 @@
+//Prevayler(TM) - The Free-Software Prevalence Layer.
+//Copyright (C) 2001-2003 Klaus Wuestefeld
+//This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+package org.prevayler;
+
+import java.io.Serializable;
+
+/**
+ * Formal common base type for {@link Transaction} and {@link TransactionWithQuery}.
+ * Used to limit the valid types (via ...<code>? extends TransactionBase</code>...)
+ * in classes/methods/parameters referring to either of the two types using generics
+ *      
+ * @author Christian Klemke
```

Comment:
We have not been using `@author`, at least not in the `core` module, so would you mind removing this line? You will still be credited as the Git commit author.

Reply:
Actually, that one wasn't on purpose; the tag is part of my default comment code for new types template in Eclipse. I'll remove it.

Reply:
Removed.